docs/reference/workflow-reference.md

```diff
@@ -1,3 +1,491 @@
+
+A workflow is a sequence of data blocks and processing blocks. It defines an order of operations that start with a data block, which may be followed by up to five processing blocks.
+
+```python
+workflow = up42.initialize_workflow(
+    project_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    workflow_id="7fb2ec8a-45be-41ad-a50f-98ba6b528b98",
+)
+```
+
+## Projects
+
+### max_concurrent_jobs
+
+The `max_concurrent_jobs` attribute returns the maximum number of jobs that can run simultaneously.
+
+The returned format is `int`.
+
+<h5> Example </h5>
+
+```python
+workflow.max_concurrent_jobs
+```
+
+## Workflows
+
+### info
+
+The `info` attribute returns metadata of a specific workflow.
+
+The returned format is `dict`.
+
+<h5> Example </h5>
+
+```python
+workflow.info
+```
+
+### update_name()
+
+The `update_name()` function allows you to change the name and description of a workflow.
+
+```python
+update_name(
+    name,
+    description,
+)
+```
+
+<h5> Arguments </h5>
+
+| Argument      | Overview                              |
+| ------------- | ------------------------------------- |
+| `name`        | **str**<br/>The workflow name.        |
+| `description` | **str**<br/>The workflow description. |
+
+<h5> Example </h5>
+
+```python
+workflow.update_name(
+    name="Sentinel-2 with tiling",
+    description="Implements tiling on free Sentinel-2 data",
+)
+```
+
+### delete()
+
+The `delete()` function allows you to delete a workflow.
+
+```python
+delete()
+```
+
+<h5> Example </h5>
+
+```python
+workflow.delete()
+```
+
+## Workflow tasks
+
+Workflow tasks are blocks that are added to a workflow. A workflow task uses a specific block version that specifies its input JSON parameters and what blocks can be added before and after it.
+
+### workflow_tasks
+
+The `workflow_tasks` attribute returns a list of workflow tasks in a workflow.
+
+The returned format is `dict[str, str]`.
+
+<h5> Example </h5>
+
+```python
+workflow.workflow_tasks
+```
+
+### get_workflow_tasks()
+
+The `get_workflow_tasks` function returns a list of workflow tasks in a workflow.
+
+```python
+get_workflow_tasks(basic)
+```
+
+The returned format is `Union[list, dict]`.
+
+<h5> Arguments </h5>
+
+| Argument | Overview                                                                                                                                                                                                                     |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `basic`  | **bool**<br/>Determines how to return a list of workflow tasks:</br/><ul><li>`True`: return only simplified task names and block versions.</li><li>`False`: return the full response.</li></ul>The default value is `False`. |
```

Comment:
1. Let's move type to a separate column
2. </br/> breaks the layout and is not needed